Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang
```

Comment:
this is too huge image to your program, find another way to how can you minimize size of container.